gapis/api/vulkan/replay.go

```diff
@@ -34,6 +35,10 @@ import (
 	"github.com/google/gapid/gapis/stringtable"
 )
 
+var (
+	lenientDeviceMatching = flag.Bool("lenient-device-matching", false, "_Makes replay device matching more lenient")
+)
+
```

Comment:
Shouldn't this be in `cmd/gapis/main.go` ?

Reply:
No. Think about the library dependencies.